server/conf/application.conf

```diff
@@ -20,9 +20,16 @@ jira.baseUrl = ${?JIRA_BASE_URL}
 jira.username = ${?JIRA_USERNAME}
 jira.password = ${?JIRA_PASSWORD}
 
+
+luis.url = "https://api.projectoxford.ai/luis/v2.0/apps/76383aa1-8f28-4d9d-88ff-620755082531"
+luis.appId = "76383aa1-8f28-4d9d-88ff-620755082531"
+luis.subscription-key = "b2f3a7eb209943109116a76b0cdf4adf"
```

Comment:
Should these `luis` configuration properties be secret? If they are sensitive information, please remove as Github is a public repo. Use ENV variable instead, please refer to https://www.playframework.com/documentation/2.5.x/ProductionConfiguration#Using-environment-variables for details.